chapters/challengeflag.adoc

```diff
@@ -0,0 +1,14 @@
+== Challenge Flag
+
+A challenge flag allows teams to challenge a decision of the referee:
+
+. If referees decision was correct, team loses a timeout.
+. If referees decision was incorrect, the correct decision is applied and the team doesn't lose a timeout.
+
+NOTE: In both cases the team loses one flag.
+
+Only one ruling may be challenged at a time.
+
+Each team will receive two flags at the start of the game.
```

Comment:
It should be added that a challenge can only be requested, if the team has at least one timeout left.